src/raftstore/store/store.rs

```diff
@@ -72,6 +72,21 @@ type Key = Vec<u8>;
 const MIO_TICK_RATIO: u64 = 10;
 const PENDING_VOTES_CAP: usize = 20;
 
+#[derive(Clone)]
+pub struct Engines {
+    pub engine: Arc<DB>,
```

Comment:
if we use kv_engine, raft_engine, then Store::new will have 8 arguments, clippy will warning
~~~
warning: this function has too many arguments (8/7)
   --> src/raftstore/store/store.rs:190:5
    |
190 | /     pub fn new(ch: StoreChannel,
191 | |                meta: metapb::Store,
192 | |                cfg: Config,
193 | |                engines: Arc<DB>,
...   |
242 | |         Ok(s)
243 | |     }
    | |_____^
    |
    = note: #[warn(too_many_arguments)] on by default
    = help: for further information visit https://github.com/Manishearth/rust-clippy/wiki#too_many_arguments
~~~

do we need a clippy.toml config?   or use tupple instead?